src/features/targeting/README.md

```diff
@@ -9,12 +9,13 @@ Targeting Api is provided for each message. It can be consumed by the below pseu
 function SharedLinkPreviewModal() {
   const { getTargetingApi } = useContext(TargetingContext);
 
-  const { shouldShow, onClose, onSeen } = getTargetingApi('adhoc_shared_link_preview');
+  const { shouldShow, onClosed, onSeen } = getTargetingApi('adhoc_shared_link_preview');
```

Comment:
i tend to prefer `toClose` as it's the verb and makes sense in this context. realize it's a little weird with `onSeen` (which is also a verb, but in the past tense) but we're doing `toClose` in the present tense.

Reply:
i'm sorry, i meant to say `onClose` (not `toClose`). that was a typo on my part.

Reply:
hi, I think toClose is better in terms of it is clear this should be called hand. 
shouldShow() // boolean to show
onSeen() // call after seen
toClose() // call before close

Reply:
fixed per review